src/videojs-contrib-hls.js

```diff
@@ -134,6 +135,39 @@ export default class HlsHandler extends Component {
       externHls: Hls
     });
 
+    this.masterPlaylistController_.on('loadedmetadata', () => {
+      let mediaGroups = this.masterPlaylistController_.masterPlaylistLoader_.master.mediaGroups;
+      let media = this.masterPlaylistController_.masterPlaylistLoader_.media();
+      let attributes = {
+        audio: mediaGroups.AUDIO[media.attributes.AUDIO] || {main: {default: true}},
+      };
+      let audioTracks = this.tech_.audioTracks();
+      // clear current audioTracks
+      let i = audioTracks.length;
+      while (i--) {
+        audioTracks.removeTrack(audioTracks[i]);
+      }
+
+      for (let label in attributes.audio) {
+        let hlstrack = attributes.audio[label];
+        let kind = 'alternative';
+        let enabled = hlstrack.default || false;
+        let language = hlstrack.language || '';
+        if (enabled) {
+          kind = 'main';
+        }
+
+        let track = new AudioTrack({
```

Comment:
This might just be a style preference (so feel free to ignore), but I think it would look cleaner (and save variables) as:

```javascript
audioTracks.addTrack(new AudioTrack({
  kind: hlstrack.default ? 'main' : 'alternative',
  enabled: hlstrack.default || false,
  language: hlstrack.language || '',
  label,
});
```